web/client/components/data/featuregrid/FeatureEditorFallback.jsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useMemo } from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import { getDockSize } from '../../../selectors/featuregrid';
+import { mapLayoutValuesSelector } from '../../../selectors/maplayout';
+import Loader from '../../misc/Loader';
+
+const FeatureEditorFallback = ({ size, dockStyle }) => {
+    const containerStyle = useMemo(() => {
+        return { height: `${size * 100}%`, ...dockStyle };
+    }, [size, dockStyle]);
+    return (
+        <div className="feature-editor-fallback-container" style={containerStyle}>
+            <Loader size={100} style={{ margin: '0 auto' }} />
+        </div>
+    );
+};
+
+export default connect(createSelector(
+    getDockSize,
```

Comment:
this is an exception usually I would ask to use the selector but in this way it should load less dependencies in this file.
```
state => state?.featuregrid?.dockSize
```

Reply:
Done